boards/arduino-atmega-common/include/periph_conf.h

```diff
@@ -139,6 +141,29 @@ extern "C" {
 #endif
 /** @} */
 
+/**
+ * @name ADC configuration
+ *
+ * The number of ADC channels of the atmega328p depends on the package:
+ *  - 6-channel 10-bit ADC in PDIP package
+ *  - 8-channel 10-bit ADC in TQFP and QFN/MLF package
+ * Arduino UNO / Duemilanove has thereby 6 channels. But only 5 channels can be
+ * used for ADC, because the pin of ADC5 emulate a software triggered interrupt.
+ *  -> boards -> arduino-atmega-common -> include -> board_common.h
+ *
+ * @{
+ */
+#ifdef CPU_ATMEGA328P
+#define ADC_NUMOF       (5)
+#define ADC_PIN_OFFSET  14
+#endif
+
+#ifdef CPU_ATMEGA2560
+#define ADC_NUMOF       (16)
+#define ADC_PIN_OFFSET  54
+#endif
```

Comment:
And also: do I see it correctly, that `ADC_PIN_OFFSET` is not used anywhere anymore?!